quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/mappers/PropertyMappers.java

```diff
@@ -83,9 +84,10 @@ public static ConfigValue getValue(ConfigSourceInterceptorContext context, Strin
         //
         // The special handling of log properties is because some logging runtime properties are requested during build time
         // and we need to resolve them. That should be fine as they are generally not considered security sensitive.
+        // If however expressions are not enabled that means quarkus is specifically looking for runtime defaults, and we should not provide a value
```

Comment:
This seems a bit like a fragile assumption but we'd probably noticed soon enough if that ever changed.

Reply:
We'd either need to not allow our logging options to not affect build time or we'd need a deeper hook into quarkus to differentiate between logging config used at build time and logging config for runtime defaults.